cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -771,6 +772,8 @@ spec:
                       type: object
                       nullable: true
                       x-kubernetes-preserve-unknown-fields: true
+                priorityClassName:
```

Comment:
do we need this to place under `gateway:`?

Reply:
Yes, the intention is for it to be under `gateway`

Reply:
I moved this under `gateway`